docs/user_guides/guardrails-library.md

```diff
@@ -788,6 +789,38 @@ define flow
   bot provide report answer
 ```
 
+### Cleanlab
+
+NeMo Guardrails supports using the [Cleanlab Trustworthiness Score API](https://cleanlab.ai/blog/trustworthy-language-model/) as an output rail (you need to have the `CLEANLAB_API_KEY` environment variable set).
+
+```yaml
+rails:
+  output:
+    flows:
+      - cleanlab trustworthiness
+```
+
+The `cleanlab trustworthiness` flow uses trustworthiness score with a default threshold of 0.5 to determine if the output should be allowed or not (i.e., if the trustworthiness score is below the threshold, the response is considered "untrustworthy").
```

Comment:
Should this value be 0.6 instead?

Reply:
thanks for catching that! i've updated the doc.